docs/tools/appium/continuous-integration/jenkins_gradle.md

```diff
@@ -39,6 +39,8 @@ These values would then be sent through the appropriate DesiredCapabilities obje
     capabilities.setCapability(TestObjectCapabilities.TESTOBJECT_DEVICE, deviceId);
 {% endhighlight %}
 
+For more information check out our [Appium setup guide.](https://help.testobject.com/docs/tools/appium/setups)
```

Comment:
remove the line, the link has to be in some other location (maybe further up, when you are hinting at the basic/watcher setup)